sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -785,6 +785,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val PARQUET_META_CACHE_ENABLED = buildConf("spark.sql.parquet.metadataCache.enabled")
+    .doc("To indicate if enable parquet file meta cache, it is recommended to enabled " +
+      "this config in long-running process mode, such as Thrift Server, default is false")
```

Comment:
Is this recommendation `true` when the files are updated frequently (maybe hourly) and Spark accesses the files only once (query daily)?
> it is recommended to enabled this config in long-running process mode, such as Thrift Server